packages/react-i18n/src/e2e/server.test.tsx

```diff
@@ -0,0 +1,117 @@
+import '../test/matchers';
```

Comment:
I haven't seen this `e2e` convention used in our other packages. We usually write tests for the external API of our packages anyhow, so I'd just put this in the regular `tests` folder.